src/bundle/Resources/public/scss/_custom.scss

```diff
@@ -3,31 +3,38 @@ $ez-admin-transition: cubic-bezier(.25, .8, .25, 1);
 
 /* eZ colors */
 $ez-color-primary: #f15a10;
-$ez-color-primary-hover: #cf4c0c;
-$ez-color-secondary: #0f6d95;
-$ez-color-secondary-hover: #0c5472;
+$ez-color-primary-hover: darken($ez-color-primary, 15%);
+$ez-color-secondary: #106d95;
+$ez-color-secondary-hover: darken($ez-color-secondary, 15%);
 $ez-color-hyperlink: #0645ad;
 $ez-color-hyperlink-hover: #032863;
 $ez-color-positive: #00825c;
 $ez-color-warning: #f7d000;
-$ez-color-danger: #cb2540;
-$ez-color-danger-hover: #b32032;
+$ez-color-danger: #d92d42;
+$ez-color-danger-hover: darken($ez-color-danger, 15%);
 
 $ez-white: #fff;
 $ez-ground-base-pale: #fafafa;
-$ez-ground-base-light: #f3f3f3;
-$ez-ground-base: #e5e3e3;
-$ez-ground-base-medium: #ced4da; // color is not in guidelines (currently never used)
-$ez-ground-base-dark: #bbbbbb;
-$ez-color-base-pale: #878787;
-$ez-color-base-light: #6b6b6b;
-$ez-color-base-medium: #4b4b4b;
-$ez-color-base-dark: #333333;
-$ez-black: #000;
+$ez-ground-base-light: #f5f5f5;
+$ez-ground-base: #f3f3f3;
+$ez-ground-base-medium: #ededed;
+$ez-ground-base-dark: #e5e3e3;
+$ez-color-base-pale: #dbdbdb;
+$ez-color-base-light: #878787;
+$ez-color-base-medium: #646464;
+$ez-color-base-dark: #555;
+$ez-black: #333;
 
+$ez-secondary-ground-pale: #e1f5ff;
```

Comment:
It makes more sense to group `ez-ground-primary*` and `ez-secondary-ground*`
And also better to be consistent on naming: `ground-primary` or `primary-ground` / `ground-secondary` or `secondary-ground`. :)

Reply:
We are basically arranging colors in three groups: ez colors (branding) and grays & blues palettes (for background composition). Your comment would be related to the third group. I reused existing variables and when needing a new one I have followed the current labeling logic.

+1 to consistency on naming. We should tackle this too.

Reply:
Ping @dew326 & @sunpietro What do you think?

Reply:
The naming here is a little bit misleading. I would expect that `ground-primary` is orange and `ground-secondary` is blue but it seems all to be blues. I think we should use one common name and add some suffix.

Reply:
@inakijv could you explain why all colors are blue variants?

Reply:
@lserwatka for the composition of the background, we are using both gray and blue hues. The former group is used for the background composition. The latter adds the soft accent to that background (as well as the some of the interactions) in order to give all the stage for the brand (and more saturated) colors that we selected.

The screenshot below with most of the colors listed out (based on `_custom.scss` file can help to explain this.
![ui_colors_update](https://user-images.githubusercontent.com/9256718/33452576-ae5a8ba8-d5e0-11e7-858a-9f6368c8bd69.png)